lib/constants.py

```diff
@@ -18,7 +18,7 @@
 import os
 
 # Images which are OSTree based
-OSTREE_IMAGES = ["fedora-coreos", "rhel4edge"]
+OSTREE_IMAGES = ["centos-9-bootc", "fedora-coreos", "rhel4edge"]
```

Comment:
We could make this a bit more vendor neutral and call it `IMMUTABLE_IMAGES`. But that might require changes in Cockpit as well

Reply:
Yeah, lots of them. I kept it because (1) it's referenced in a lot of places, and (2) decorators like `@skipOstree` are applied for lots of different reasons: "/usr is not writable", or "our coreos image doesn't have PCP support", etc.

For now, bootc images are still ostree, and will be for a long time. It just changes the transport of images/updates and the CLI for applying them.